airflow_dags/open_elections/ca.py

```diff
@@ -0,0 +1,63 @@
+from airflow_dags.open_elections.load_by_state import StateMetadata, load_to_dolt
+import argparse
+
+PRECINCT_VOTES_CREATE = '''
+    CREATE TABLE ca_precinct_votes (
+        `election_id` VARCHAR(256),
+        `precinct` VARCHAR(256),
+        `party` VARCHAR(256),
+        `candidate` VARCHAR(256),
+        `votes` INT,
+        `mail` INT,
+        `election_day` INT,
+        `absentee` INT,
+        `reg_voters` INT,
+        `total_ballots` INT,
+        `early_votes` INT,
+        `provisional` INT,
+        `polling` FLOAT,
+        PRIMARY KEY (`election_id`, `precinct`, `party`, `candidate`)
+    )
+'''
+
+PRECINCT_VOTE_PKS = ['election_id', 'precinct', 'party', 'candidate']
+
+PRECINCT_VOTES_COLS = [
+    'election_id',
+    'precinct',
+    'party',
+    'candidate',
+    'votes',
+    'mail',
+    'election_day',
+    'absentee',
+    'reg_voters',
+    'total_ballots',
+    'early_votes',
+    'provisional',
+    'polling',
+]
+
+COUNTY_VOTES_CREATE = '''
```

Comment:
Did we not prove this was a view?

Reply:
I don't think so, it could be the case that for some states and counties precinct level data is not yet available, and we should bias towards making data available. I do want to do some data validation stuff.